### DISCUSSION TITLE
GitHub App bypass branch protection rules & status checks

### DISCUSSION THREAD

**Discussion by renescheepers** [community/community#34583] [Pull Requests]
I'm currently working on an GitHub App that needs to be able to push into a protected branch that also requires status checks to pass. The status checks cause an error when trying to push into the protected branch. Is there an option I'm not seeing or this a missing feature?

**Comment by neotrow** [comment]
+1 

Apps can now be listed as an exception for branch protected branches that need a PR but not for the status checks which seems odd.

**Comment by ffried** [comment]
This is probably the same feature request: https://github.com/orgs/community/discussions/18145

**Reply by seanstory** [reply]
There's a workaround listed on that thread, using Rulesets. :success:
